test/e2e/apps/daemon_restart.go

```diff
@@ -51,7 +51,7 @@ import (
 // 1. Daemons restart automatically within some sane time (10m).
 // 2. They don't take abnormal actions when restarted in the steady state.
 //	- Controller manager shouldn't overshoot replicas
-//	- Kubelet shouldn't restart containers
+//	- kubelet shouldn't restart containers
```

Comment:
Just wondering if this is the correct syntax for `kubelet` then we may need to update the same for `Controller manager` and `Scheduler` as well? or it'll be okay to use `Kubelet` as part of comments here?

Reply:
In my opinion, it doesn't matter whether you use `kubelet` or `Kubelet` on the annotation, it's just a handy modification here. Mainly think that the writing method of `kubelet` on the log output should be unified with other places to start with lowercase letters.